openjdk_regression/playlist.xml

```diff
@@ -63,9 +76,15 @@
 			<subset>SE80</subset>
 			<subset>SE90</subset>
```

Comment:
I've already added SE100 to this playlist in master, so you will have to merge conflicts.  We will need to repeal SE100 from (I think) the hotspot tests, based on Muneer's note in slack about certain tests in Java10 needing additional commandline options (in which case we will need separate <test> in playlist for those, so we'll need to co-ordinate.